docs/support-tasks.md

```diff
@@ -29,6 +29,21 @@ $ bundle exec rake manage:change_email_address[<old_email_address>, <new_email_a
 
 [change]: https://deploy.blue.production.govuk.digital/job/run-rake-task/parambuild/?TARGET_APPLICATION=email-alert-api&MACHINE_CLASS=email_alert_api&RAKE_TASK=manage:change_email_address[from@example.org,to@example.org]
 
+## View subscriber's recent emails
+
+This task shows the most recent email delivery attempts made to the given user.
+It takes two parameters: `email_address` (required), and `limit` (optional).
+`limit` defaults to 10, but you can override this if you need to see more of
+the user's history.
+
+```bash
+$ bundle exec rake manage:view_emails[<email_address>,<limit>]
+```
+
+[⚙ Run rake task on production][view_emails]
+
+[view_emails]: https://deploy.blue.production.govuk.digital/job/run-rake-task/parambuild/?TARGET_APPLICATION=email-alert-api&MACHINE_CLASS=email_alert_api&RAKE_TASK=manage:view_emails[email@example.org]
```

Comment:
```suggestion
[view_emails]: https://deploy.blue.production.govuk.digital/job/run-rake-task/parambuild/?TARGET_APPLICATION=email-alert-api&MACHINE_CLASS=email_alert_api&RAKE_TASK=manage:view_emails["email@example.org"]
```

Nitpick: Should the email address be in quotes?

Reply:
It works without quotes, and the other rake task links don't have quotes in the URL either.

Thanks for the review :)